scripts/.spelling_failures

```diff
@@ -0,0 +1,10 @@
+content/en/blog/_posts/2016-05-00-Coreosfest2016-Kubernetes-Community.md
+content/en/blog/_posts/2016-10-00-Dynamic-Provisioning-And-Storage-In-Kubernetes.md
+content/en/blog/_posts/2017-02-00-Postgresql-Clusters-Kubernetes-Statefulsets.md
```

Comment:
👋 @yuswift .
This file marks blog files that contain spelling errors?

Reply:
yes